tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -405,9 +406,29 @@ def _submit_batch_job(
         else:
             return self._remote_command_executor.run_remote_command(submission_command, raise_on_error=raise_on_error)
 
+    def _dump_job_output(self, job_info):
+        match = re.match(r".*StdErr=(.*)?\sStdIn=(.*)?\sStdOut=(.*)?\sPower.*", job_info)
+        if match:
+            stderr = match.group(1)
+            stdout = match.group(3)
+            if stdout == stderr:
+                command = f'echo "stderr/stdout:" && cat {stderr}'
+            else:
+                command = f'echo "stderr:" && cat {stderr} && echo "stdout:" && cat {stdout}'
+            result = self._remote_command_executor.run_remote_command(command)
+
+            return result.stdout
+
```

Comment:
It would be nice to log something when the function is unable to parse the StdOut or StdErr paths from the job info string.

Reply:
Done